pkg/distributor/distributor_test.go

```diff
@@ -1467,6 +1467,244 @@ func TestDistributor_Push_HistogramValidation(t *testing.T) {
 	}
 }
 
+func TestDistributor_SampleDuplicateTimestamp(t *testing.T) {
+	const (
+		metricName         = "series"
+		ts1        int64   = 10
+		ts2        int64   = 20
+		ts3        int64   = 30
+		ts4        int64   = 40
+		v1         float64 = 1
+		v2         float64 = 2
+		v3         float64 = 3
+		v4         float64 = 4
```

Comment:
[nit] What's the purpose of defining constants of constants? Can't we just use the actual values in the test? :)

Reply:
I have removed the constants